Assets/AppCenter/Plugins/AppCenterSDK/Auth/Shared/UserInformation.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AppCenter.Unity.Auth.Internal;
+
+namespace Microsoft.AppCenter.Unity.Auth
+{
+#if UNITY_IOS || UNITY_ANDROID
+    using RawType = System.IntPtr;
+#else
+    using RawType = System.Type;
+#endif
+
+    public class UserInformation
+    {
+        public string AccountId;
```

Comment:
BTW why it's public _now_? (since it's not a property)

Reply:
I was referring to the Models in Crashes (like ErrorAttachmentLog and ErrorReport ), and looks like there instance variables are public, so I set them to be public as well (Sorry I'll change the getter and setter format).

Reply:
Let's change this to properties. We have all of it as properties.